pymongoexplain/explainable_collection.py

```diff
@@ -16,20 +16,25 @@
 from typing import Union, List, Dict
 
 import pymongo
+from pymongo.collection import Collection
 from bson.son import SON
 
 from .commands import AggregateCommand, FindCommand, CountCommand, \
     UpdateCommand, DistinctCommand, DeleteCommand, FindAndModifyCommand
 
 Document = Union[dict, SON]
 
+
 class ExplainCollection():
     def __init__(self, collection):
         self.collection = collection
         self.last_cmd_payload = None
 
     def _explain_command(self, command):
         command_son = command.get_SON()
+        if command_son == {}:
+            self.last_cmd_payload = {}
+            return {}
```

Comment:
This should be removed. It's impossible for a command to be empty (`{}`).

Reply:
Done.